tests/stdlib/ttypeinfo_std.nim

```diff
@@ -0,0 +1,60 @@
+import typeinfo
```

Comment:
should be named: tests/stdlib/ttypeinfo.nim

(shouldn't conflict with 
tests/cpp/ttypeinfo.nim
tests/cpp/ttypeinfo2.nim
, nim allows multiple modules with same name)
worst case, tests/cpp/ttypeinfo.nim itself could be renamed to tests/cpp/ttypeinfo1.nim

Reply:
I will try that later.